Reload window after changing password

Hafizzle opened the pull request.

Hafizzle committed:
Reload window after changing password

SabreCat commented on website/client/src/pages/settings/settingRows/passwordSetting.vue:
```diff
@@ -168,6 +168,7 @@ export default {
 
         this.passwordUpdates = {};
+        window.location.reload();
         this.$store.dispatch('snackbars:add', {
```
Hmm... with a `reload()` happening immediately before this, the user won't see the snackbar about a successful password change. I'll start a conversation in Slack about what we want for this.

Hafizzle committed:
Shows password change success message

Displays a success snackbar after a user successfully changes their password. This provides visual confirmation to the user that the password update was successful.
The success message is displayed only once after the page reloads.

Hafizzle committed:
lint fix trailing spaces
